source/manual/pagerduty.html.md

```diff
@@ -47,22 +46,22 @@ anyone else being phoned.
 
 To trigger the drill manually, follow these steps:
 
-1. SSH onto the monitoring box:
+1. Exec onto the alertmanager box after logging onto the production cluster:
```

Comment:
Can we elaborate on logging into production cluster? I get an error when I try to run it:
```
Error from server (Forbidden): pods "alertmanager-kube-prometheus-stack-alertmanager-0" is forbidden: User "aga.dufrat-poweruser" cannot get resource "pods" in API group "" in the namespace "monitoring"
```

I've done:
- `eval $(gds aws govuk-production-poweruser -e --art 8h)`
- `k config use-context production`

Reply:
hmmm, I wonder if special permissions are needed to access it?

Reply:
I've updated the notes to say that for manual triggering of pagerduty we should ask on the #platform-engineering channel.